src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+class ExcavatorProducer implements MachineProducer<Excavator> {
```

Comment:
The `ExcavatorProducer` class should be public to ensure that it can be accessed from outside of its package.